app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -16,6 +16,8 @@ def index
                             client_id: @authorize_form.client_id,
                             errors: @authorize_form.errors.messages)
 
+      return create if already_allowed?
```

Comment:
Do we have to go through the whole `create` action, which will perform validations and link the identity again, or can we redirect directly to `redirect_uri`?